encryptedlogging/src/main/kotlin/com/automattic/encryptedlogging/EncryptedLogging.kt

```diff
@@ -25,6 +27,11 @@ public interface EncryptedLogging {
      */
     public fun resetUploadStates()
 
+    /**
+     * A method for the client to use to observe the upload result of the encrypted logs.
+     */
+    public fun observeEncryptedLogsUploadResult(): StateFlow<OnEncryptedLogUploaded?>
```

Comment:
I don't think we have to expose `StateFlow` - it's an implementation detail.

```suggestion
    override fun observeEncryptedLogsUploadResult(): Flow<OnEncryptedLogUploaded?> {
```

Reply:
You're right, will do! 👍

Reply:
Done: 25195ccc2713d0b506881cee8258a120cab2dd2f @wzieba